shared/constants/snaps/permissions.ts

```diff
@@ -15,11 +15,11 @@ export const EndowmentPermissions = Object.freeze({
 } as const);
 
 // Methods / permissions in external packages that we are temporarily excluding.
-export const ExcludedSnapPermissions = Object.freeze({
-  eth_accounts:
+export const ExcludedSnapPermissions = Object.freeze({});
+
+export const ExcludedSnapEndowments = Object.freeze({
+  'endowment:caip25':
```

Comment:
I lack some context around how/where we are translating between `eth_accounts` requests to the new format at runtime. (I might have just not read that part of the PR yet).

But we should make sure that `eth_accounts` remains blocked if it is valid in `wallet_requestPermissions` or in permission granting. It is currently blocked because it is not supported as a requested permission at install-time. It can still be granted at runtime.